velox/exec/fuzzer/DuckQueryRunner.h

```diff
@@ -46,6 +46,18 @@ class DuckQueryRunner : public ReferenceQueryRunner {
   /// Assumes that source of AggregationNode or Window Node is 'tmp' table.
   std::optional<std::string> toSql(const core::PlanNodePtr& plan) override;
 
+  /// Returns the name of the values node table in the form t_<id>.
+  std::string getTableName(const core::ValuesNodePtr& valuesNode);
+
+  // Traverses all nodes in the plan and returns all tables and their names.
+  std::unordered_map<std::string, std::vector<velox::RowVectorPtr>>
+  getAllTablesAndNames(const core::PlanNodePtr& plan);
```

Comment:
nit: I think calling it getAllTables would be just as well..

Reply:
Done.